apis/apps/v1alpha1/opsrequest_webhook.go

```diff
@@ -261,8 +275,28 @@ func (r *OpsRequest) validateReconfigure(cluster *Cluster) error {
 	if reconfigure == nil {
 		return notEmptyError("spec.reconfigure")
 	}
+	if len(reconfigure.Configurations) == 0 {
```

Comment:
this can be validated with validation rules `+kubebuilder:validation:XValidation:rule`

Reply:
done